package.json

```diff
@@ -111,6 +115,7 @@
   },
   "devDependencies": {
```

Comment:
We need to start cleaning `dependencies` and `devDependencies` from the main package.
For instance `inquirer` is used only while migration and carries a weight of [`815.9 kB`](https://bundlephobia.com/result?p=inquirer@5.1.0).

We should better shift dependencies under the packages it is required and lighten up `webpack-cli` package.

Reply:
as long as you move them into a separate repo, I think this doesn't make any sense. replacing `inquirer` for CLI is not advisable AFAICT.

Reply:
+1 @sendilkumarn